### PR TITLE
Add Klingon, Catalan and Quenya Lang

### PR DESCRIPTION
**Pull request by PurpleNienna**

### CODE REVIEW

File: lang/lang-catala.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	module.exports.vortaro = function() {
+		return {
+			'1': ' vegades hem dit Holiiiiiiiiiiiiiiii! Putos malalts...',
+			'2': 'Hi',
+			'3': 'hi',
+			'4': 'Holiiiiiiiiiiiiiiii!!',
+			'5': 'A tu no et faig ni puto cas',
+			'6': 'Club obert',
+			'7': 'Club tancat',
+            '8': 'No queden mes taules per tirar',
```
Comment (wolframtheta) [inline lang/lang-catala.js]: Would be nice to put aligned to the other numbers

Comment (wolframtheta) [inline lang/lang-catala.js]: Would be nice to put aligned to the other numbers

File: lang/lang-catala.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+            '9': 'No caben mes taules al despatx'
```
Comment (wolframtheta) [inline lang/lang-catala.js]: Same

File: lang/lang-klingon.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+	module.exports.vortaro = function() {
+		return {
+			'1': ' poH qavan maqarmeH! petaQ...',
+			'2': 'bIpIv\'a\'',
+			'3': 'bIpIv\'a\'',
+			'4': 'nuqneeeeeeH',
+			'5': 'SoHvaD pagh  \'e\' vIQoy.',
+			'6': 'jeqqIj poS',
+			'7': 'jeqqIj SoQ',
+            '8': 'pagh raS tu\'lu\'',
```
Comment (wolframtheta) [inline lang/lang-klingon.js]: Same

File: lang/lang-klingon.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+            '9': 'raSDaq logh pagh tu\'lu\''
```
Comment (wolframtheta) [inline lang/lang-klingon.js]: Same

File: lang/lang-quenya.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+	module.exports.vortaro = function() {
+		return {
+			'1': ' lú emme quet Alatulyaaaaaaa! Nátyë necindo...',
+			'2': 'Alla',
+			'3': 'alla',
+			'4': 'Alatulyaaaaaaaaaa',
+			'5': 'Áva quetë',
+			'6': 'Ampano palime',
+			'7': 'Ampano cumna',
+            '8': 'Lala tare pálala',
```
Comment (wolframtheta) [inline lang/lang-quenya.js]: Same

File: lang/lang-quenya.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+            '9': 'Lala tare ilmen'
```
Comment (wolframtheta) [inline lang/lang-quenya.js]: Same

**Review by wolframtheta — CHANGES_REQUESTED** [review]
Would be nice to put aligned to the other numbers

**Review by PurpleNienna — COMMENTED** [review]
Tabs

**Pull request closed without merging**